docs: add documentation for svelte.dev (#31)
sveltejs/mcp: documentation/docs/30-capabilities/10-tools.md

---
title: Tools
---

This is the list of available tools provided by the MCP server. The list of tools is given to the model which can decide to call one or more of the tools during a session.

## list-sections

Provides a list of all the available documentation sections and their use cases if available. The titles of the sections will be used by the LLM to get up-to-date documentation directly from the official documentation.

## get-documentation

> Allows the model to get the full documentation for the requested sections. The documentation is fetched on the fly from the official Svelte documentation and it will always be up to date.

Reviewer: ```suggestion
Allows the model to get the full (and up-to-date) documentation for the requested sections directly from [svelte.dev/docs](/docs).
```
